src/libcore/iter/traits.rs

```diff
@@ -836,6 +837,52 @@ macro_rules! float_sum_product {
 integer_sum_product! { i8 i16 i32 i64 i128 isize u8 u16 u32 u64 u128 usize }
 float_sum_product! { f32 f64 }
 
+#[stable(feature = "iter_arith_traits_option", since="1.29.0")]
```

Comment:
Should be 1.28.0 😄

Reply:
Heh, just planning ahead. I'm sure the decision process will delay this at least 4 weeks given that it's insta-stable.